server/tabby_server/services/resource_format.py

```diff
@@ -93,50 +106,58 @@ def google_books_test_all_books():
             # Dict to hold the individual book attributes
             entry = {}
 
+            # Only keeps the entry if the isbn could be found. Only keeps the
+            # isbn 13.
             try:
-                entry['isbn'] = items[i]["volumeInfo"]["industryIdentifiers"]
+                # Purely to get under the Flake8 character limit.
+                ii = "industryIdentifiers"
```

Comment:
Although it's more verbose, I think it's clearer to leave the strings as-is instead of replacing them with a shorter variable. This way, it won't be confused with the loop variable `i`.

Reply:
I completely agree, but if I did, flake8 would become unhappy as that would exceed the character limits. Since this was just a test, I didn't feel the need to do anything special and just went quick with what worked.

Reply:
Yes, but the `black` formatter would take care of lines over 79 by splitting it over multiple lines. It is a bit uglier if it did, so since this is for testing purposes, this is fine.